apis/infrastructure/v1/cert_types.go

```diff
@@ -17,7 +18,8 @@ type CertificateSpec struct {
 	// is provided by the user. Allowed values are:
 	// * SelfSigned: A certificate is going to be generated using an own private key.
 	// * Provided: Pre-existence of the TLS Secret (see SecretName) with a valid certificate is assumed.
-	// +kubebuilder:validation:Enum=SelfSigned;Provided
-	// +kubebuilder:default=SelfSigned
+	// * DefaultIngress: Default ingress certificate configured for OpenShift
+	// +kubebuilder:validation:Enum=SelfSigned;Provided;DefaultIngress
+	// +kubebuilder:default=DefaultIngress
```

Comment:
if we change the default value, it can cause problem for these users did not set it in old release.

Reply:
I think this should be okay. By default users will use default certs. I guess we need to document this change. @israel-hdez Can you confirm?

Reply:
yes  defaultIngress would be the default one for users. If we document it, it would be enough.

Reply:
Yep, if there is an existing DSCI/DSC, it won't be updated. Currently this has been introduced in dsc